tests/dialects/test_snowflake.py

```diff
@@ -240,11 +240,18 @@ def test_timestamps(self):
             },
         )
         self.validate_all(
-            "SELECT DATE_PART(month FROM a::DATETIME)",
+            "SELECT DATE_PART(month, a::DATETIME)",
```

Comment:
This was changed to reflect DATE_PART's syntax as shown in the [docs](https://docs.snowflake.com/en/sql-reference/functions/date_part.html#syntax).